packages/eslint-plugin-lit-a11y/docs/rules/scope.md

```diff
@@ -1,7 +1,7 @@
 # scope
 
-Enforce scope attribute is only used on &lt;th&gt; elements.
-The scope attribute may only be used on `<th>` elements.
+Enforce scope attribute is only used on &lt;th&gt; elements and on custom elements.
```

Comment:
See the comment down below, we should adjust this text, I dont think we even have to mention custom elements here, and can just keep it as it was 🙂